modules/Officials/host/EasyAbp.WeChatManagement.Officials.Host.Shared/FodyWeavers.xml

```diff
@@ -0,0 +1,3 @@
+﻿<Weavers xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="FodyWeavers.xsd">
+  <ConfigureAwait />
```

Comment:
Should replace all the `<ConfigureAwait />` to `<ConfigureAwait ContinueOnCapturedContext="false" />`.